web/ui/templates/status.html

```diff
@@ -60,7 +60,7 @@ <h2 id="targets">Targets</h2>
                 </span>
               </td>
               <td>
-                <span class="cursor-pointer" data-toggle="tooltip" title="" data-original-title="Before relabeling: {{.MetaLabels}}">
+                <span class="cursor-pointer" data-toggle="tooltip" title="" data-html=true data-original-title="<b>Before relabeling:</b>{{range $k, $v := .MetaLabels}}<br>{{$k | html | html}}=&quot;{{$v | html | html}}&quot;{{end}}">
```

Comment:
It works either way, but it doesn't need to be escaped, unescaping `<b>` (something already not escaped) results in the same as unescaping `&lt;b&gt;`.